apps/super-app/src/common/components/navbar/NavBar.css

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css?family=Hind");
+a:visited {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
+  gap: 56px;
+  position: absolute;
```

Comment:
This won't matter now, but let's drop the absolute positioning. It won't work well when we're positioning multiple elements on the page and I don't think it makes a difference to the demo.